src/js/components/Chart/doc.js

```diff
@@ -80,6 +80,18 @@ export const doc = Chart => {
       Related to 'overflow', this allows control over how much space
       is available for bars and points to overflow into.`,
     ),
+    point: PropTypes.oneOf([
+      'circle',
+      'diamond',
+      'square',
+      'star',
+      'triangle',
+      'triangleDown',
+    ]).description(
+      `When using a 'point' type, what shape the points should use.
+      If not specified, 'square' will be used if not 'round',
+      otherwise 'circle' will be used if 'round' is true.`,
+    ),
     round: PropTypes.bool
       .description('Whether to round the line ends.')
       .defaultValue(false),
```

Comment:
It is not clear that the `round` prop also affect the `point` type, the description should mention that round also "rounds" the default square shape of point to a circle, or even better, be explicit that `round` triumph any explicit `point` value with returning a `circle` point, so users won't wonder how `round` should behave when used in congestion with `star`, `triangle`(s) or `square`.

Reply:
I will attempt to clarify more in the documentation. Just so you know here, `point` trumps `round`. `round` only comes into play if `point` is specified.

Reply:
documentation updated